.github/workflows/pr-test-e2e.yml

```diff
@@ -105,6 +105,14 @@ jobs:
         if: always()
         run: kubectl --context kind-kubeflex get deployments -A
 
+      - name: Dump kubeflex-controller-manager logs
+        if: always()
+        run: |
+          echo "--- Dumping logs for previous kubeflex-controller-manager container ---"
+          kubectl logs -n kubeflex-system deploy/kubeflex-controller-manager -p || echo "INFO: No previous container found."
+          echo "--- Dumping logs for current kubeflex-controller-manager container ---"
+          kubectl logs -n kubeflex-system deploy/kubeflex-controller-manager || echo "ERROR: Could not dump current logs."
```

Comment:
The `|| echo "ERROR: Could not dump current logs."` is not really helpful. Anyone looking at the job log will see what did and did not happen.